odkx-src/docs-tech-guide.rst

```diff
@@ -139,7 +139,7 @@ and the :ref:`ODK-X Forum <join-forum>`
 
    It is better to ask a question in the "wrong" venue
    than to not ask the question at all.
-   Many of the same people are present in all three places,
+   Many of the same people are present in both the places,
```

Comment:
minor edit: both places rather than both the places

Reply:
I first wrote both places and later changed it to both the places, really weak at this :'(
I will fix this.